g2core/board/ArduinoDue/gShield-pinout.h

```diff
@@ -258,12 +258,12 @@ pin_number kADC14_PinNumber = -1;  // Not physially pinned out
 
 
 // GRBL / gShield compatibility pins -- Due board ONLY
-
+// NOTE: These dont appear to work
```

Comment:
Good catch. These should be removed, and inputs made on these pins and preconfigured with the correct actions.